pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/agent/impl/DockerPipelineFromDockerfileScript.groovy

```diff
@@ -72,15 +72,16 @@ class DockerPipelineFromDockerfileScript extends AbstractDockerPipelineScript<Do
 
     private Closure buildImage() {
         return {
+            def dockerfilePath = describable.getDockerfilePath(script.isUnix())
```

Comment:
assuming this is called when actually running on the node.

Reply:
It is. I double checked. =) All of this script is run within the node block.